packages/python3.rb

```diff
@@ -3,33 +3,28 @@
 class Python3 < Package
   description 'Python is a programming language that lets you work quickly and integrate systems more effectively.'
   homepage 'https://www.python.org/'
-  @_ver = '3.9.2'
+  @_ver = '3.9.4'
   version @_ver
   license 'PSF-2.0'
   compatibility 'all'
   source_url "https://www.python.org/ftp/python/#{@_ver}/Python-#{@_ver}.tar.xz"
-  source_sha256 '3c2034c54f811448f516668dce09d24008a0716c3a794dd8639b5388cbde247d'
+  source_sha256 '4b0e6644a76f8df864ae24ac500a51bbf68bd098f6a173e27d3b61cdca9aa134'
 
   binary_url({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.2-chromeos-x86_64.tar.xz'
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/python3-3.9.4-chromeos-x86_64.tar.xz'
   })
   binary_sha256({
-    aarch64: '83b292381a6fcda63f3c8c6c4b682694be673b42ed14eb1482e5cfe6f1c0f835',
-     armv7l: '83b292381a6fcda63f3c8c6c4b682694be673b42ed14eb1482e5cfe6f1c0f835',
-       i686: '8f3ba22ea8824a958aafe9127e8fa078406323a6d9785e2986d3dc4ac45f671a',
-     x86_64: 'd3195034ad9d26e9e4c0c08c35bff6f551cc5661b970284c96deb81f32d34fb8'
+    aarch64: 'fe4fde3f9738ac76096dfdc13ff6931543e2f612035421e294f87d2bfc09773c',
+     armv7l: 'fe4fde3f9738ac76096dfdc13ff6931543e2f612035421e294f87d2bfc09773c',
+       i686: 'a9e7b61911679cda55f65cf5ca29076ee2ac4540c8dd7bda5322df66d4d6e4bd',
+     x86_64: 'cd3a63250e76cd887a620df69f37d349d4cea0d39e3d8bc2ceef9400a40befbb'
   })
 
-  depends_on 'bz2'
-  depends_on 'xzutils'
-  depends_on 'sqlite'
-  depends_on 'krb5'
-  depends_on 'libtirpc'
```

Comment:
Are these packages included in core by other core packages?  We have to be careful about removing dependencies of core packages.

Reply:
These are the actual runtime deps which are in core:
```
  depends_on 'glibc'
  depends_on 'readline'
  depends_on 'ncurses'
  depends_on 'openssl'
  depends_on 'libdb'
  depends_on 'gdbm'
  depends_on 'zlibpkg'
  depends_on 'sqlite'
  depends_on 'bz2'
  depends_on 'xzutils'
  depends_on 'libffi'
```
As build deps, sqlite, krb5, libtirpc, and xzutils are all already in core.

Since Python3 is IN core, should I put all the build and runtime deps back in?

Reply:
That's what I did.

Reply:
The core packages and their dependencies are what make up the core packages so if you remove any dependencies from one core package, another core package better install those dependencies too or else the list is not complete.  You can test easy enough with a fresh install to see if it matches `tools/core_packages.txt`.

Reply:
Have the binaries been updated?

Reply:
I haven't touched the binaries. There are no changes in the compile, so I don't think they need updating.

Reply:
Also, the simple solution to issues with core packages getting installed via some hierarchy of package installs... would be to just create an initial hierarchy of core installs... and just make that in initial install. in install.sh. (That's what I have been doing using `tsort`.)

Reply:
Here's my current workaround for getting all of core installed:
```
# Use parallel to download essential packages, then install.
corefilepkgs=$(curl -Ls https://github.com/skycocker/chromebrew/raw/master/tools/core_packages.txt )
buildessentialpkgs=$(grep "^[[:blank:]] depends_on" packages/buildessential.rb | awk '{print $2}' | sed "s;';;g" )
echo "$EARLY_PACKAGES $corefilepkgs $buildessentialpkgs less most" | tr '\n' ' ' > /tmp/sorted.txt
for i in {1..7}
do
  echo "Dependency Search Pass $i"
  [[ -f /tmp/depraw.txt ]] && rm /tmp/depraw.txt
  get_deps
done
cat /tmp/sorted.txt
# Don't re-download EARLY_PACKAGES
for i in $EARLY_PACKAGES
do
  sed -i "/$i/d" /tmp/sorted.txt
done
sorted=$(cat /tmp/sorted.txt)
rm /tmp/{sorted.txt,deps.txt,depraw.txt}
echo "corefiles & buildessentials sorted in 1 level dependency order: $sorted"
parallel -j 4 CURL="$CURL" crew -d download -- "$sorted" || true
for i in  libjpeg gdk_pixbuf $sorted
do
yes |  CURL=$CURL crew install $i || true
done
```